src/google/cloud/ndb/_datastore_query.py

```diff
@@ -33,6 +33,7 @@
 
 MoreResultsType = query_pb2.QueryResultBatch.MoreResultsType
 MORE_RESULTS_TYPE_NOT_FINISHED = MoreResultsType.Value("NOT_FINISHED")
+MORE_RESULTS_AFTER_LIMIT = MoreResultsType.Value("MORE_RESULTS_AFTER_LIMIT")
```

Comment:
To follow the pattern exactly established by MORE_RESULTS_TYPE_NOT_FINISHED, it would have to be MORE_RESULTS_TYPE_MORE_RESULTS_AFTER_LIMIT, which just seems like overkill. But we can do that if you think we should.

Reply:
Oh you're right, that doesn't sound great. Let's keep it as is